src/pages/storycreator.page.tsx

```diff
@@ -0,0 +1,23 @@
+import { Container } from '@mui/material';
+import { useRouter } from 'next/router';
+
+import StoryCreator from '@/features/storycreator/StoryCreator';
+import StoryOverview from '@/features/storycreator/StoryOverview';
+
+export default function StoryCreatorPage(): JSX.Element {
```

Comment:
as a follow-up: let's just split this into two routes:
`/src/pages/storycreator/index.page.tsx` for the overview
`/src/pages/storycreator/[id].page.tsx` for a specific story